packages/recommend/src/types/WithRecommendMethods.ts

```diff
@@ -9,6 +9,16 @@ import { TrendingFacetsQuery } from './TrendingFacetsQuery';
 import { TrendingItemsQuery } from './TrendingItemsQuery';
 import { TrendingQuery } from './TrendingQuery';
 
+/**
+ * This MultipleQueriesResponse type differs from MultipleQueriesResponse imported from @algolia/client-search as it omits "SearchForFacetValuesResponse" in results
+ */
+export type MultipleQueriesResponse<TObject> = {
```

Comment:
this should have a different name and probably a test to ensure it's caught by ci

Reply:
Renamed to `RecommendQueriesResponse`

Reply:
this RecommendQueriesResponse shouldn't be used also in javascript client: `packages/client-search/src/methods/client/multipleQueries.ts` ?
because #1460 had impact also on this method